src/dust/coagulation/coagulation.cpp

```diff
@@ -0,0 +1,296 @@
+//========================================================================================
+// (C) (or copyright) 2023-2024. Triad National Security, LLC. All rights reserved.
+//
+// This program was produced under U.S. Government contract 89233218CNA000001 for Los
+// Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+// for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+// in the program are reserved by Triad National Security, LLC, and the U.S. Department
+// of Energy/National Nuclear Security Administration. The Government is granted for
+// itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works, distribute copies to
+// the public, perform publicly and display publicly, and to permit others to do so.
+//========================================================================================
+
+// Artemis includes
+#include "dust/coagulation/coagulation.hpp"
+#include "artemis.hpp"
+#include "dust/dust.hpp"
+#include "geometry/geometry.hpp"
+
+namespace Dust {
+namespace Coagulation {
+//----------------------------------------------------------------------------------------
+//! \fn  StateDescriptor Coagulalation::Initialize
+//! \brief Adds intialization function for coagulation package
+std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin, Params &dustPars) {
+  auto coag = std::make_shared<StateDescriptor>("coagulation");
```

Comment:
Seems to me that this just belongs in the `dust` package. Thoughts @pdmullen @adamdempsey90?